docs/fundamentals/executor/executor-api.md

```diff
@@ -294,6 +294,30 @@ Text: hello world
 ```
 
 
+You can even pull an `Executor` from the jina hub and use it directly as a python object.
```

Comment:
I have a suggestion, but I am not 100% sure it's the best; you can try it out and see.
To make the structure a bit clearer, we could use tabs for the different ways of spawning an Executor outside a Flow. That would combine the old part about Executors being a regular Python object, and your new explanation.
Something like this:

`Executor` objects can be used directly, just like any regular Python object.
There are two ways of instantiating an Executor object: From a local Python class, and from the Jina Hub.

[tab] From local Python
...

[tab] From Jina Hub
...

Btw, is there also a way to create an Executor directly form YAML? Then that could be a third tab.